src/Microsoft.Azure.Functions.PowerShellWorker.csproj

```diff
@@ -21,7 +21,7 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
 
   <ItemGroup>
     <PackageReference Include="Grpc.Net.Client" Version="2.52.0" />
-    <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.2.13" />
+    <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.2.18" />
```

Comment:
Please upgrade `.Net SDK 6` to version `6.0.418`.

Reply:
Done